.circleci/config.yml

```diff
@@ -175,6 +174,60 @@ jobs:
     <<: *defaults
     <<: *test_steps
 
+  test-e2e:
+    machine:
+      image: ubuntu-2204:2023.02.1
+    environment:
+      - WP_ENV_VERSION: "8.0.0"
```

Comment:
Maybe as a follow PR (since this also needs to be dealt in the develop repo), we could have that in `.p4-env.json` and "extract" that value from there so we have only one place to update this when needed.

Reply:
right, I'm also thinking about adding it as a local package, since it can now [use local package in priority](https://github.com/WordPress/gutenberg/pull/50980)